### DISCUSSION TITLE
Integrating Flow with Azure release pipeline

### DISCUSSION THREAD

**Discussion by INLANA01** [postmanlabs/postman-flows#206] [Q&A] [answered]
I wanted to create Flows for performing end-to-end testing of the APIs we use, and previously we had collection runner which I could use in Azure release pipeline with the help of Newman. Is there any way I could use Flow for the same reason, so that I could create a pipeline block (using Flows) for end-to-end testing as a part of out deployment process?

**Comment by saswatds** [comment, marked as answer]
Hey @INLANA01 we are currently working on a spec by which you can use one Flow inside another Flow like a block. With this you will be able to create a Azure Pipeline block and then re-use it across various other flows.
